spec/helpers/stylesheet_helper_spec.rb

```diff
@@ -30,6 +30,12 @@
           visible: :all,
         )
       end
+
+      it 'adds preload header without nopush attribute' do
+        render_stylesheet_once_tags
+
+        expect(response.headers['link']).to eq('</stylesheets/styles.css>; rel=preload; as=style')
+      end
```

Comment:
maybe make the spec indicate what shouldn't be there?
```suggestion
      it 'adds preload header without nopush attribute' do
        render_stylesheet_once_tags

        expect(response.headers['link']).to eq('</stylesheets/styles.css>; rel=preload; as=style')
        expect(response.headers['link']).to_not include('nopush')
      end
```

Reply:
Sure. My brain rejects assertions of both strict equality and then also inclusion that's not present anyways in the strictly-equal, but I think that communicates intent a bit better.